css/card.css

```diff
@@ -101,3 +394,345 @@
+  --color1: #7894ff90;
+}
+
+.details-dialog-card:before,
+.details-dialog-card:after, 
+#gacha-controls .card-image:before,
+#gacha-controls .card-image:after {
+  content: "";
+  position: absolute;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  top: 0;
+  background-repeat: no-repeat;
+  opacity: 0;
+  mix-blend-mode: color-dodge;
+  transition: all 0.33s ease;
+  border-radius: 5%/3.5%;
+}
+
+.details-dialog-card:before, #gacha-controls .card-image:before {
+  background-position: 50% 50%;
+  background-size: 300% 300%;
+  background-image: linear-gradient(115deg, transparent 0%, var(--color1) 25%, transparent 47%, transparent 53%, var(--color2) 75%, transparent 100%);
+  filter: brightness(0.5) contrast(1);
+  z-index: 1;
+}
+
+.details-dialog-card.basic:before, #gacha-controls .card-image.basic:before {
+  filter: brightness(0) contrast(0);
+}
+
+.details-dialog-card.holo:before, .details-dialog-card.ultra:before, .details-dialog-card.secret:before,
+#gacha-controls .card-image.holo:before, #gacha-controls .card-image.ultra:before, #gacha-controls .card-image.secret:before {
+  background-image: linear-gradient(115deg, transparent 20%, var(--color1) 36%, var(--color2) 43%, var(--color3) 50%, var(--color4) 57%, var(--color5) 64%, transparent 80%);
+}
+
+.details-dialog-card:after, #gacha-controls .card-image:after {
+  background-blend-mode: overlay;
+  z-index: 2;
+  filter: brightness(1) contrast(1);
+  transition: all 0.33s ease;
+  mix-blend-mode: color-dodge;
+}
+
+.details-dialog-card.basic:after, #gacha-controls .card-image.basic:after {
+  background-image: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHhtbG5zOnhsaW5rPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5L3hsaW5rIiB3aWR0aD0iNTAwIiBoZWlnaHQ9IjUwMCI+CjxmaWx0ZXIgaWQ9Im4iPgo8ZmVUdXJidWxlbmNlIHR5cGU9ImZyYWN0YWxOb2lzZSIgYmFzZUZyZXF1ZW5jeT0iLjciIG51bU9jdGF2ZXM9IjEwIiBzdGl0Y2hUaWxlcz0ic3RpdGNoIj48L2ZlVHVyYnVsZW5jZT4KPC9maWx0ZXI+CjxyZWN0IHdpZHRoPSI1MDAiIGhlaWdodD0iNTAwIiBmaWxsPSIjMDAwIj48L3JlY3Q+CjxyZWN0IHdpZHRoPSI1MDAiIGhlaWdodD0iNTAwIiBmaWx0ZXI9InVybCgjbikiIG9wYWNpdHk9IjAuMyI+PC9yZWN0Pgo8L3N2Zz4=);
+  background-position: center;
+  background-size: 200%;
+  background-blend-mode: overlay;
+  mix-blend-mode: screen;
+}
+
+.details-dialog-card.holo:after, #gacha-controls .card-image.holo:after {
+  --space: 4%;
+  --galaxybg: center 521px;
+  background-image: url(../images/galaxy.jpg), url(../images/galaxy.jpg), url(../images/galaxy.jpg), repeating-linear-gradient( 82deg, rgb(219, 204, 86) calc(var(--space)*1), rgb(121, 199, 58) calc(var(--space)*2), rgb(58, 192, 183) calc(var(--space)*3), rgb(71, 98, 207) calc(var(--space)*4), rgb(170, 69, 209) calc(var(--space)*5), rgb(255, 90, 180) calc(var(--space)*6), rgb(255, 90, 180) calc(var(--space)*7), rgb(170, 69, 209) calc(var(--space)*8), rgb(71, 98, 207) calc(var(--space)*9), rgb(58, 192, 183) calc(var(--space)*10), rgb(121, 199, 58) calc(var(--space)*11), rgb(219, 204, 86) calc(var(--space)*12) );
```

Comment:
I'm especially curious how to read this line. How were the numbers chosen? How are they affected when the card moves in 3d?

Reply:
These numbers are taken directly from the simey stuff; so the specifics of why they are chosen I don't really know. It's just creating a layered background (images and then gradients), which then uses a number of blend effects. It's easier to see the gradient by itself when there is no image applied.